### ISSUE TITLE
Basic Densifier output doesn't work

### ISSUE THREAD

**Issue by UnknownSerhan** [MJRLegends/ExtraPlanets#749]
### ---Issue Report---
**(PLEASE DONT DELETE THIS TEMPLATE, YOUR ISSUE WILL BE IGNORED AND CLOSED IF YOU DO. | Required)**

**Have you checked Closed Issues (if applicable & PLEASE CHECK BEFORE POSTING | Required):** 
Yes, they have been all closed because there wasn't enough information.

### Description of Issue(Required)
There is no way to output the content of a basic densifier. None of the transfer methods work.

### Steps to Reproduce Issue(Required)
1. Place down a basic densifier and process some fluid with it.
2. Try to output the produced items automatically. The bottom is supposed to work but it doesn't. None of the sides work with Mekanism logistical transporters on pull mode as well.

**Crash log/Log (if applicable):** [PASTEBIN. Do NOT copy and paste the log into the issue post itself. See below on how to get them]
> *How do i locate a crash log* see https://docs.mjrlegends.com/General%20Minecraft/crash-log/

**Screenshots (if applicable):** 
![2023-03-01_22 04 12](https://user-images.githubusercontent.com/69000972/222254156-79418318-0191-4d8e-aa12-d1b96e528cc3.png)
![2023-03-01_22 04 27](https://user-images.githubusercontent.com/69000972/222254158-de352e78-811c-401b-9fdc-a8081e63748a.png)
![2023-03-01_22 04 39](https://user-images.githubusercontent.com/69000972/222254161-fd7fd771-f7a9-4824-9274-6bd683e417ff.png)

**Mod List (Required):** 
There are a lot of mods, but this basically happened with just galacticraft, the addons and a few extra mods too.

**Version of Mod using (PLEASE DONT USE 'Latest' OR 'Newest' | Required):** [VERSION]
Minecraft 1.12.2, ExtraPlanets 0.7.5

**Where did this bug happen(Required):** [Single Player or Server]
Happens on both singleplayer and server.

### Additional Information
[Any other information that may be able to help me with the problem]

**Comment by MJRLegends** [comment]
Hey sorry the delay in a reply,

Please can you try placing a hopper below the machines and trying to hook up your pipe to suck items from the hopper instead?

**Comment by UnknownSerhan** [comment]
I don’t think that will change anything, the machine can’t output to anything including hoppers. But sure I can try it.

**Comment by mikehammack** [comment]
I just tested that, hopper placed directly below the machine and the coal still stays in the densifier.

**Comment by UnknownSerhan** [comment]
Apparently the item output is made for the battery slot and not for the machine output. I can output the batteries from the machine by using mekanism logistical transporters on pull mode (hopper below the denfisier doesn't output the battery, actually the bottom does nothing, I can output the battery from behind the densifier)
![2023-03-07_23 49 04](https://user-images.githubusercontent.com/69000972/223560891-2568b8b9-72ae-49fa-811e-a327600a74c0.png)
So the issues are the item output is made for wrong slot and bottom side does nothing even though it is an output slot.

Edit: Just tested, I can output the battery from any side except the bottom.

**Comment by UnknownSerhan** [comment]
So the input/output sides work for batteries for every side except the bottom. The bottom is supposed to output the machine product but I guess it is missing the code or maybe it is broken.

**Comment by MJRLegends** [comment]
Hey thank you for your bug report & replies,

I've fixed the issue ready for the next update

**Comment by MJRLegends** [comment]
MC 1.12.2 Update have released https://www.curseforge.com/minecraft/mc-mods/extraplanets/files/4466264
MC 1.11.2 Update have released https://www.curseforge.com/minecraft/mc-mods/extraplanets/files/4466275
MC 1.10.2 Update have released https://www.curseforge.com/minecraft/mc-mods/extraplanets/files/4466284

MC 1.8.9 update are delayed due a issue. I will keep this open in till other versions have released
